cmd/checkpoint-installer/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
```

Comment:
We shouldn't need a separate image just for the install script. Just add `install.sh` to the checkpoint docker image, and the runtime command usage can just be different (`command: /install.sh` vs `command: /checkpoint`)